src/main/org/codehaus/groovy/transform/SortableASTTransformation.java

```diff
@@ -112,7 +113,7 @@ private static void implementComparable(ClassNode classNode) {
         }
     }
 
-    private static Statement createCompareToMethodBody(List<PropertyNode> properties) {
+    private static Statement createCompareToMethodBody(List<PropertyNode> properties, Boolean reversed) {
```

Comment:
`boolean` as the parameter type would be consistent with the other methods below.